server/immich-openapi-specs.json

```diff
@@ -5816,12 +5816,14 @@
         "type": "object",
         "properties": {
           "name": {
-            "type": "string"
+            "type": "string",
+            "description": "Person name."
+          },
+          "featureFaceAssetId": {
```

Comment:
What about primary asset id? This id is also used for matching against right?

Reply:
This is the asset to crop the face of the person from

Reply:
Unless I am mistaken the API doesn't ever reference asset_face, AssetFaceEntity, or faces ever. That is an implementation detail. All the external client knows is that a person has a thumbnail and then can update it with an asset id. Personally, I think calling it something with "face" makes me think you are referring to the asset face table and that seems like an implementation detail.

Reply:
It does here https://github.com/immich-app/immich/blob/f21b9ac4892ea091b2783d6b38ecd95994e20ab9/server/src/infra/repositories/person.repository.ts#L80-L82